api/scripts/update-codegen.sh

```diff
@@ -72,6 +75,24 @@ function generate::core {
           cp -R "${OUTPUT_BASE}/${MODULE}/." "${SCRIPT_ROOT}"
 }
 
+function generate:crds() {
+          cd "${SCRIPT_ROOT}/.."
+          OUTPUT_BASE=$(mktemp -d)
+          trap 'rm -rf "${OUTPUT_BASE}"' ERR EXIT
+
+          "${GOPATH}/bin/controller-gen" crd paths=./api/core/v1alpha2/... output:crd:dir="${OUTPUT_BASE}"
+
+          while IFS= read -r -d '' file
+            do
+              # TODO: legacy, delete after all crds are auto-generated
+              if ! [[ " ${ALLOWED_RESOURCE_GEN_CRD[*]} " =~ [[:space:]]$(yq '.spec.names.kind' "$file")[[:space:]] ]]; then
```

Comment:
You declare ALLOWED_RESOURCE_GEN_CRD as an array of strings, but there is only one item with multiple kinds. And here you convert array to one string. ALLOWED_RESOURCE_GEN_CRD can be just a string with space separated kinds.

Reply:
fixed